src/connector.js

```diff
@@ -901,6 +898,17 @@ function updateQuerySummaryState( newState ) {
 		querySummaryElement.textContent = "";
 		querySummaryElement.innerHTML = resultErrorTemplateHTML;
 	}
+
+	// Focus to results section
+	if( !querySummaryState.isLoading && querySummaryElement.textContent !== "" ) {
+		let focusElement = resultsSection.querySelector( "h2" );
+		
+		if( focusElement ) {
+			focusElement.role = "status";
```

Comment:
This is problematic, applying the role "status" on an "h2" element remove the heading level 2 semantic.

Please apply the role status on an inner span or on a wrapper div. Ideally the role status would already been defined prior the change and not every time there is a change.

See the SC statement of 4.1.3. It mentions "without receiving focus".

Anyway, the role status can't be applied to the "h2" element
```suggestion
```

Reply:
The `role="status"` was removed since focus is going diretly to the H2 after a search action, which satisfies the criterion.